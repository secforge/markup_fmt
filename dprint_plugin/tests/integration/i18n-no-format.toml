"vue.custom_block" = "none"
