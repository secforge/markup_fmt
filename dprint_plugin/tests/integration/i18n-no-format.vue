<template>
  <div>{{ $t('hello') }}</div>
</template>

<i18n lang="json">
{
"en":    {"hello":   "Hello"},
  "de": {"hello":"Hallo"   }
}
</i18n>
