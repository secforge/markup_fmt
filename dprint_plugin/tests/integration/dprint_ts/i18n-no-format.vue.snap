---
source: dprint_plugin/tests/integration.rs
assertion_line: 123
---
<template>
  <div>{{ $t("hello") }}</div>
</template>

<i18n lang="json">
{
"en":    {"hello":   "Hello"},
  "de": {"hello":"Hallo"   }
}
</i18n>
